packages/libgrib1/package.py

```diff
@@ -35,40 +35,29 @@ class Libgrib1(MakefilePackage):
 
     version('master', branch='master')
     version('22-01-2020', commit='3d3db9a9a090f6798c2fd4290c271dd58ff694e0')
-    
-    variant('slave', default='tsa', description='Build on slave tsa, daint or kesch', multi=False)
-
-    depends_on('mpi')
-
-    def setup_environment(self, spack_env, run_env):
-        spack_env.set('LIBNAME', 'grib1')
+   
+    def edit(self, spec, prefix):
```

Comment:
Probably not optimal to do this in `edit`, but it's probably ok too.

Reply:
Not doing it in edit is not read by spack...